### DISCUSSION TITLE
Hello World

### DISCUSSION THREAD

**Discussion by nwg-piotr** [nwg-piotr/nwg-shell#14] [General]
Have an idea, a problem, or would you like to share your opinion about nwg-shell? Feel free to do it here. 

**Please start new threads in appropriate categories.**

**Comment by 31JSON** [comment]
Great job on a nice shell for sway.

I'm one of those (seemingly rare) people who prefers a light theme. Is there any chance of getting a built in light/dark theme selector, or a more central way of selecting colors? The colors of various parts of the shell are spread out over many config files.

**Reply by nwg-piotr** [reply]
Thank you!

Well, I was thinking about it, but css style sheets are the most flexible solution, and yes, not really comfortable. I could add fields to edit some arbitrarily selected properties, but what to do about the rest? Also: making a one, centralized css file for all components would be good from the point of view of nwg-shell users, but this is only available on Arch Linux, while separate components have been packaged for various distros. Those are tough decisions to make.
